config/default/config_split.config_split.sitenow_ccom.yml

```diff
@@ -0,0 +1,16 @@
+uuid: db56db91-c318-49db-9498-1069e8d767a3
+langcode: en
+status: false
+dependencies: {  }
+id: sitenow_ccom
```

Comment:
For this split and the module, I'm wondering about prefixing with `sitenow_`? Our module/split naming is all over the place, so it's hard to find a clear best practice. In this case, these modifications are specific to a group of sites and would never be turned on for sites outside of that group. I thinking that maybe `ccom` for the split name and `ccom_core` for the module, since that is consistent with how we name other similar modules.